allennlp/models/crf_tagger.py

```diff
@@ -132,6 +133,9 @@ def forward(self,  # type: ignore
         logits = self.tag_projection_layer(encoded_text)
         predicted_tags = self.crf.viterbi_tags(logits, mask)
 
+        # Just get the tags and ignore the score.
+        predicted_tags = [x for x, y in predicted_tags]
```

Comment:
our typechecker is for sure going to complain about this, because you're reassigning with a different type, maybe change it to something like

```
predicted_tags_and_scores = ...

predicted_tags = [tags for tags, _ in predicted_scores]
```

Reply:
Oops, yea I ran the other tests locally but forgot to run mypy. I'll make that fix!